### PR TITLE
Client side diagnostics

### PR DESCRIPTION
**Pull request by yuce**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -17,3 +19,6 @@ test:
 
 test-all:
 	go test -tags=integration
+
+release:
+	printf "package pilosa\nconst Version = \"$(VERSION)\"" > version.go
```
[the remark this answers is not in the document]
Comment (yuce) [reply Makefile]: `version.go` with the current version is committed starting with this PR. There's no build time for the client so we can't use the same mechanism with the Pilosa server.

The library releaser would run `make release` which produces the `version.go` and commit that file and then push to master. I can't see any other way which would work. But if you have another suggestion, I can implement that.

File: README.md [hunk 2]
```diff
@@ -534,6 +534,13 @@ If you are using a self signed certificate, just pass `pilosa.TLSConfig(&tls.Con
 client, _ := NewClient("https://01.pilosa.local:10501", TLSConfig(&tls.Config{InsecureSkipVerify: true}))
 ```
 
+## Diagnostics
```
[the remark this answers is not in the document]
Comment (yuce) [reply README.md]: updated

File: Makefile (hunk 1 above)
[the remark this answers is not in the document]
Comment (codysoyland) [reply Makefile]: Oh, I understand now. I didn't realize that file was in version control - I thought I generated it when testing. That all makes sense!

**Comment by yuce** [comment]
Great!

**Pull request merged**